soa/service/remote_credential_provider.cc

```diff
@@ -1,188 +0,0 @@
-// This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
```

Comment:
As it is, users have no ways to configure an external credential provider.  If you feel that this is required than we would need to add a way to configure it and document clearly what the REST interface of that provider is.  My feeling is that if someone has this level of expertise he may as well modify the MLDB code to get what he wants.  Any strong feeling here?